build.sbt

```diff
@@ -136,19 +52,19 @@ lazy val scalaClasses = (project in file("scala"))
     libraryDependencies ++= Seq(
       "org.apache.thrift" % "libthrift" % thriftVersion,
       "com.twitter" %% "scrooge-core" % scroogeVersion,
-      "org.scalacheck" %% "scalacheck" % "1.14.0" % "test"
+      "org.scalacheck" %% "scalacheck" % "1.17.0" % "test"
     )
   )
 
 lazy val thrift = (project in file("thrift"))
-  .settings(commonSettings)
+  .settings(artifactProductionSettings)
   .disablePlugins(ScroogeSBT)
   .settings(
     name := "content-entity-thrift",
     description := "Content entity model Thrift files",
     crossPaths := false,
-    packageDoc / publishArtifact := false,
-    packageSrc / publishArtifact := false,
+    packageDoc / publishArtifact := false, //do we need to remove this? also because if root is using "publish / skip := true" instead?
+    packageSrc / publishArtifact := false, //do we need to remove this? also because if root is using "publish / skip := true" instead?
```

Comment:
Interesting - it looks like we've never published the `thrift` project as an artifact, so actually it would be right to put  `publish / skip := true` on this submodule too!